test/Npgsql.Tests/CopyTests.cs

```diff
@@ -22,7 +22,7 @@ public async Task Issue2257()
             if (IsMultiplexing)
                 Assert.Ignore("Multiplexing: fails");
 
-            using (var conn = OpenConnection(new NpgsqlConnectionStringBuilder(ConnectionString) { CommandTimeout = 3 }))
+            using (var conn = OpenConnection(new NpgsqlConnectionStringBuilder(ConnectionString) { CommandTimeout = 30 }))
```

Comment:
30 is the default, so this can just do a default OpenConnection like other tests, no?